packages/teams-js/src/public/media.ts

```diff
@@ -25,17 +25,14 @@ import {
   validateSelectMediaInputs,
   validateViewImagesInput,
 } from '../internal/mediaUtil';
-import {
-  callCallbackWithErrorOrResultFromPromiseAndReturnPromise,
-  callCallbackWithSdkErrorFromPromiseAndReturnPromise,
-  generateGUID,
-  InputFunction,
-} from '../internal/utils';
+import { generateGUID } from '../internal/utils';
+import { runtime } from '../public/runtime';
 import { FrameContexts, HostClientType } from './constants';
 import { ErrorCode, SdkError } from './interfaces';
-import { runtime } from './runtime';
 
 /**
+ * @deprecated
+ *
  * @alpha
```

Comment:
Do we want to keep the @alpha tag? It suggests that this namespace is intended to be used by third-party developers eventually but has not yet been released.

Reply:
No, the @alpha tag is not supported in our documentation system. Please remove it. I'll be making a separate change to remove @alpha everywhere else.